src/Node.php

```diff
@@ -354,7 +354,7 @@ public function jsonSerialize() {
     }
 
     /**
-     * Get the end index of a Node.
+     * Get the end position of a Node.
```

Comment:
index (=offset) is more precise than position. Position can also mean (line, column).